docs/contributing/index.md

```diff
@@ -12,26 +15,28 @@ Please start by reading the other docs in this folder to understand how the mono
 
 You should see the reactotron electron app open on your machine with the development menu open.
 
-### Step 2: Create an Ignite react-native app to test your changes
+### Step 2: Use the example react-native app to test your changes
+
+1. Start the example app with `yarn start:example`
+2. Launch the app in the simulator of your choice.
+3. You should see output in the reactotron app.
 
-1. Initialize a new [ignite app](https://github.com/infinitered/ignite) onto your machine. `npx ignite-cli new PizzaApp --yes`. This new app will already be set up to use reactotron.
-2. Once this app is done initializing, start it: `cd PizzaApp && yarn start`.
-3. Launch the app in the simulator of your choice.
-4. You should see output in the reactotron app.
+### Step 3: Making changes
 
-### Step 3: Making changes to `reactotron-app`
+Changes made to the main reactotron electron app in `./apps/reactotron-app` will be reflected in the reactotron app that you started in step 1. You may need to kill the app and restart it to see your changes.
 
-Any changes you make to the main reactotron electron app that lives in `./apps/reactotron-app` will be reflected in the reactotron app that you started in step 1. You may need to kill the app and restart it to see your changes.
+[!IMPORTANT]
```

Comment:
https://github.com/orgs/community/discussions/16925

Reply:
Note that anything you put in the `docs` folder is also going to be interpreted by docusauras and published to https://docs.infinite.red/reactotron/

We should be using docusauras compatible markdown, not github markdown.